libs/designer/src/lib/core/state/operation/operationMetadataSlice.ts

```diff
@@ -161,6 +161,7 @@ export interface NodeData {
 interface AddSettingsPayload {
   id: string;
   settings: Settings;
+  ignoreDirty: boolean;
```

Comment:
Let's make this optional and default to false, just so we can only use it when we need it